### ISSUE TITLE
multiple chrome profiles for multiple items?

### ISSUE THREAD

**Issue by tiaanyun** [val92130/Supreme-Auto-Checkout#110]
I know i have to have the tabs open, but is it possible to run different tabs so i can cop multiple items faster?

**Comment by SirLovejoy** [comment]
Yes to your question! Creating multiple chrome users and installing the extension on each one is how to run the bot multiple ways .
